command/command_test.go

```diff
@@ -118,18 +138,15 @@ func testVaultServerPluginDir(tb testing.TB, pluginDir string) (*api.Client, []s
 		LogicalBackends:    defaultVaultLogicalBackends,
 		PluginDirectory:    pluginDir,
 		BuiltinRegistry:    builtinplugins.Registry,
-	})
+	}, defaultTestClusterOptions)
 }
 
 // testVaultServerCoreConfig creates a new vault cluster with the given core
 // configuration. This is a lower-level test helper.
-func testVaultServerCoreConfig(tb testing.TB, coreConfig *vault.CoreConfig) (*api.Client, []string, func()) {
+func testVaultServerCoreConfig(tb testing.TB, coreConfig *vault.CoreConfig, opts *vault.TestClusterOptions) (*api.Client, []string, func()) {
```

Comment:
This function is also used in `external-tests`. I think it would be simpler to leave this function unchanged so we don't have to worry about fixing tests that aren't relevant to this change, and instead adding a new function. For example,

```
testVaultServerCoreConfigWithOpts(...opts) {
...copy code from testVaultServerCoreConfig here and initialize the cluster with opts
}

testVaultServerCoreConfig(...) {
return testVaultServerCoreConfigWithOpts(...)
}
```

Reply:
done